gui-lib/mred/private/wx/gtk/style.rkt

```diff
@@ -96,3 +101,31 @@
 
 (define (get-label-bg-color)
   (extract-color-values (GtkStyle-bg1 the-text-style)))
+
+(define _GtkSettings (_cpointer 'GtkSettings))
+(define-gtk gtk_settings_get_default (_fun -> _GtkSettings))
+(define-gtk g_object_notify (_fun _GtkSettings _string -> _void))
+(define-gobj g_object_get/string (_fun _GtkSettings _string (r : (_ptr o _pointer)) (_pointer = #f)
+                                       -> _void
+                                       -> r)
+  #:c-id g_object_get)
+(g_object_notify (gtk_settings_get_default) "gtk-theme-name")
```

Comment:
Is this needed? I'd expect `g_object_notify` to send a notification, while the intent here generally is to receive notifications.

Reply:
Yep, as expected, removing those two lines works well.